beta/docker/README.md

```diff
@@ -0,0 +1,126 @@
+# Docker Compose v2
+
+## Prerequisites
+
+- Linux server with port 443 open
+- Docker Engine (v20.10.3+) (see [Docker Engine installation overview](https://docs.docker.com/engine/install/#server))
+- Docker Compose (v2.0+) (see [Install Docker Compose](https://docs.docker.com/compose/install/))
+
+> **Note**
+>
+> This deployment example assumes Docker Compose is run locally on the Docker
+> host. Deploying to a remote Docker host (i.e. using `docker context`) is
+> _not_ supported.
+
+## Get started
+
+1. Create a DNS record that points to your Docker host, e.g. `scim.example.com`.
+
+2. On the Docker host, clone this repository and switch to this directory:
+
+    ```sh
+    git clone https://github.com/1Password/scim-examples.git
+    cd ./scim-examples/beta/docker
+    ```
+
+3. Copy your `scimsesion` file to the working directory on the Docker host, e.g.:
+
+    ```sh
+    scp local/path/to/scimsession user@host:scim-examples/beta/docker/scimsession
+    ```
```

Comment:
Instruct user to `chmod 644 scimsession` to handle the likely scenario they only had 600,